Seagate/cortx-k8s
# CORTX-28579:  Basic regression tests for cortx-k8s

This PR introduces basic regression tests for cortx-k8s.  The goal of these tests is to implement a smoke test for cortx-k8s scripts and template files.  These tests do not introduce comprehensive system tests for CORTX.

This first release has two tests which follow the same pattern.
* TEST-DEPLOY-0001 deploys using cortx images
* TEST-DEPLOY-0002 deploys using stub (centos7) images

Each test follows this sequence:
1. Run "prereq"
2. deploy-cortx-cloud.sh
3. status-cortx-cloud.sh (confirm everything passes)
4. shutdown-cortx-cloud.sh
5. start-cortx-cloud.sh
6. destroy-cortx-cloud.sh

More checks can be added to this sequence, but at least it exercises all scripts (except upgrade).


-----
[View rendered test/cortx-k8s-tests.md](https://github.com/walterlopatka/cortx-k8s/blob/CORTX-28579_basic_regtest/test/regression/cortx-k8s-tests.md)
